### ISSUE TITLE
Mutable class attributes

### ISSUE THREAD

**Issue by cleder** [PyCQA/flake8-bugbear#57]
Mutable class attributes may lead to information leaking from one instance to another.

```
class MyClass(object):
    errors = []

    def do_something(self):
        try:
            ...
        except SomeException as e:
            self.errors.append(e)
```

Mutable class attributes should be reported similar to *B006*: Do not use mutable data structures for argument defaults.

**Comment by ambv** [comment]
This is actually hard to judge. While many of those usages are bugs, others are deliberate. Since Bugbear is a very primitive linter, it would cause too many false positives to consider this as a feature.
